app/models/job_queue.rb

```diff
@@ -99,19 +123,32 @@ def perform_job(job_execution, queue)
     end
   end
 
+  def should_queue_job?(queue)
```

Comment:
`should_` is kinda redundant since the method is already `?` -> `def queue_job?`